services/management/dbaas/version_service_client.go

```diff
@@ -196,3 +246,72 @@ func (c *VersionServiceClient) GetLatestOperatorVersion(ctx context.Context, pmm
 	latestPXCOperator, err := getLatest(pmmVersionDeps.Matrix.PXCOperator)
 	return latestPXCOperator, latestPSMDBOperator, err
 }
+
+// GetNextOperatorVersion returns operator version that is direct successor of currently installed one.
+// It returns nil if update is not available or error occurred. It does not take PMM version into consideration.
+// We need to upgrade to current + 1 version for upgrade to be successful. So even if dbaas-controller does not support the
+// operator, we need to upgrade to it on our way to supported one.
+func (c *VersionServiceClient) GetNextOperatorVersion(ctx context.Context, operatorType, installedVersion string) (nextOperatorVersion *goversion.Version, err error) {
```

Comment:
I think it's better to call this method `NextOpertorVersion` without `Get`. https://golang.org/doc/effective_go#Getters
The same with `getNext`, `getLatest`, `getLatestVersion`

Reply:
I am not sure this is getter. From my understanding you create setter/getter when you want to protect unexported fields. Whatever, as long we are consistent across codebase, I am fine with that. I am going to change it.